library/psa_crypto.c

```diff
@@ -199,7 +199,7 @@ psa_status_t mbedtls_to_psa_error( int ret )
         case MBEDTLS_ERR_CIPHER_INVALID_PADDING:
```

Comment:
About the commit message: please add the reason of the change in the commit message. Something similar as what you wrote in the PR description https://github.com/ARMmbed/mbedtls/pull/3723#issue-494221290 will be perfect.

Note: Please avoid long lines in the commit message, a quite common policy being roughly less than 70 characters.

Reply:
Thanks for the feedback. I've updated the commit message.